lib/column/string.go

```diff
@@ -52,8 +53,14 @@ func (col *String) ScanRow(dest interface{}, row int) error {
 	case *string:
 		*d = v[row]
 	case **string:
-		*d = new(string)
-		**d = v[row]
+		*d = &v[row]
+	case *[]uint8:
+		*d = []uint8(v[row])
+	case **[]uint8:
+		data := []uint8(v[row])
+		*d = &data
+	case encoding.BinaryUnmarshaler:
```

Comment:
I think it's (`encoding.BinaryUnmarshaler`) too much for a String. And you should use byte instead of uint8^^^

Reply:
So, what do you want me todo:
- Remove the case altogether? (FWIW, this is symmetric with FixedString)
- Amend the line to use []byte, instead of uint8 (although I don't understand very well why, given uint8 is used everywhere else, and they are type aliases)
- A different thing than any of the above?